op-e2e/e2eutils/challenger/helper.go

```diff
@@ -115,16 +116,18 @@ func FindMonorepoRoot(t *testing.T) string {
 	return ""
 }
 
-func applyCannonConfig(c *config.Config, t *testing.T, rollupCfg *rollup.Config, l2Genesis *core.Genesis) {
+func applyCannonConfig(c *config.Config, t *testing.T, rollupCfg *rollup.Config, l2Genesis *core.Genesis, allocType e2econfig.AllocType) {
 	require := require.New(t)
 	root := FindMonorepoRoot(t)
 	c.Cannon.VmBin = root + "cannon/bin/cannon"
 	c.Cannon.Server = root + "op-program/bin/op-program"
-	if e2eutils.UseMTCannon() {
+	if allocType == e2econfig.AllocTypeMTCannon {
 		t.Log("Using MT-Cannon absolute prestate")
 		c.CannonAbsolutePreState = root + "op-program/bin/prestate-mt.bin.gz"
-	} else {
+	} else if allocType == e2econfig.AllocTypeStandard {
 		c.CannonAbsolutePreState = root + "op-program/bin/prestate.bin.gz"
+	} else {
+		panic(fmt.Sprintf("unknown alloc type: %v", allocType))
```

Comment:
I think I'd just leave this as an `else` and default to `prestate.bin.gz` for everything else.